src/megaclient.cpp

```diff
@@ -4814,6 +4818,10 @@ void MegaClient::sc_chatupdate()
                     chat->url = ""; // not received in action packets
                     chat->ou = ou;
                     chat->title = title;
+                    if (ts != -1)
```

Comment:
Yes, that is made in purpose actually. Here, we are processing the AP. The AP will update the title, the privilege... but not the creation timestamp, which should not be reset to 0, but keep the existing one.
There are 2 cases where the action packet will effectively create the chat object: when the chat is a new one and when you are invited to an existing chat (you don't know about it before). In those cases, a new instance of `TextChat` will be created (by default, `ts=0` at ctor). The AP for those cases should include the `ts`, but other APs may not include it (currently all APs do, but it's prone to change after André's review) and we don't want to lose the existing `ts`